SPECS/pyproject-rpm-macros/pyproject-rpm-macros.spec

```diff
@@ -87,36 +107,45 @@ cp -p %{sources} .
 %install
 mkdir -p %{buildroot}%{_rpmmacrodir}
 mkdir -p %{buildroot}%{_rpmconfigdir}/azl
-install -m 644 macros.pyproject %{buildroot}%{_rpmmacrodir}/
-install -m 644 pyproject_buildrequires.py %{buildroot}%{_rpmconfigdir}/azl/
-install -m 644 pyproject_convert.py %{buildroot}%{_rpmconfigdir}/azl/
-install -m 644 pyproject_save_files.py  %{buildroot}%{_rpmconfigdir}/azl/
-install -m 644 pyproject_preprocess_record.py %{buildroot}%{_rpmconfigdir}/azl/
-install -m 644 pyproject_construct_toxenv.py %{buildroot}%{_rpmconfigdir}/azl/
-install -m 644 pyproject_requirements_txt.py %{buildroot}%{_rpmconfigdir}/azl/
+install -pm 644 macros.pyproject %{buildroot}%{_rpmmacrodir}/
+install -pm 644 pyproject_buildrequires.py %{buildroot}%{_rpmconfigdir}/azl/
+install -pm 644 pyproject_convert.py %{buildroot}%{_rpmconfigdir}/azl/
+install -pm 644 pyproject_save_files.py  %{buildroot}%{_rpmconfigdir}/azl/
+install -pm 644 pyproject_preprocess_record.py %{buildroot}%{_rpmconfigdir}/azl/
+install -pm 644 pyproject_construct_toxenv.py %{buildroot}%{_rpmconfigdir}/azl/
+install -pm 644 pyproject_requirements_txt.py %{buildroot}%{_rpmconfigdir}/azl/
+install -pm 644 pyproject_wheel.py %{buildroot}%{_rpmconfigdir}/azl/
 
 %check
-pip3 install more_itertools pytest>=3.9 toml 'tox>=3.27.1,<4.0.0'
+%if 0%{?with_check}
 export HOSTNAME="rpmbuild"  # to speedup tox in network-less mock, see rhbz#1856356
-%{python3} -m pytest -vv --doctest-modules
-test_status=$?
+%pytest -vv --doctest-modules -k "not tox"
 
 # brp-compress is provided as an argument to get the right directory macro expansion
 %{python3} compare_mandata.py -f %{_rpmconfigdir}/brp-compress
-[[ $? -eq 0 && $test_status -eq 0 ]]
+%endif
+
 
 %files
-%license LICENSE
-%doc README.md
 %{_rpmmacrodir}/macros.pyproject
 %{_rpmconfigdir}/azl/pyproject_buildrequires.py
 %{_rpmconfigdir}/azl/pyproject_convert.py
 %{_rpmconfigdir}/azl/pyproject_save_files.py
 %{_rpmconfigdir}/azl/pyproject_preprocess_record.py
 %{_rpmconfigdir}/azl/pyproject_construct_toxenv.py
 %{_rpmconfigdir}/azl/pyproject_requirements_txt.py
+%{_rpmconfigdir}/azl/pyproject_wheel.py
+
+%doc README.md
+%license LICENSE
+
+%files -n pyproject-srpm-macros
+%license LICENSE
```

Comment:
Do we even need this as a separate package? Wouldn't a `Provides:` do the trick?

Reply:
We probably don't need it, but in the interest of minimizing deltas between our specs and Fedora, (and incase something we need is added to this subpackage in the future), I plan to leave it as-is for now.